queue/src/fiskaltrust.Middleware.Localization.QueueDE/RequestCommands/DailyClosingReceiptCommand.cs

```diff
@@ -126,13 +115,65 @@ public override async Task<RequestCommandResponse> ExecuteAsync(ftQueue queue, f
             catch (Exception ex) when (ex.GetType().Name == RETRYPOLICYEXCEPTION_NAME)
             {
                 _logger.LogDebug(ex, "TSE not reachable.");
-                return await ProcessSSCDFailedReceiptRequest(request, queueItem, queue, queueDE).ConfigureAwait(false);
+
+                queueDE.DailyClosingNumber++;
+                (var masterDataChanged,var message, var type) = await UpdateMasterData(request);
+
+                actionJournals.Add(
+                    new ftActionJournal
+                    {
+                        ftActionJournalId = Guid.NewGuid(),
+                        Message = $"{message} However TSE was not reachable.",
+                        Type = $"{type:X}",
+                        ftQueueId = queue.ftQueueId,
+                        ftQueueItemId = queueItem.ftQueueItemId,
+                        Moment = DateTime.UtcNow,
+                        TimeStamp = DateTime.UtcNow.Ticks,
+                        Priority = -1,
+                        DataJson = JsonConvert.SerializeObject(new
+                        {
+                            ftReceiptNumerator = queue.ftReceiptNumerator + 1,
+                            masterDataChanged = masterDataChanged,
+                            closingNumber = queueDE.DailyClosingNumber
+                        })
+                    });
```

Comment:
I think we could reuse the `CreateClosingActionJournals` method thats also used above for creating the actionJournal.

We'd need to change the method to accept a `ulong? transactionNumber` and I think then we can use it like that 🤔 we'd then explicitly have `"transactionNumber": null` in the actionJournal as an indicator that it was not signed which could be practical when a human is reading it ^^

```suggestion
                actionJournals.Add(
                  CreateClosingActionJournals(queueItem, queue, null, masterDataChanged, $"{message} However TSE was not reachable.", type, queueDE.DailyClosingNumber));
```

Reply:
Its done